scripts/static-vdis

```diff
@@ -227,41 +221,42 @@ def connect_smapiv1_nbd(params_nbd):
 
 
 def call_backend_attach(driver, config):
-    args = map(lambda arg: arg.encode('utf8'), [driver, config])
+    args = [arg.encode('utf8') for arg in [driver, config]]
```

Comment:
Previous updated `utf8` to `utf-8`, seems this one is missed.

Reply:
There's no difference between utf8 and utf-8.
https://stackoverflow.com/questions/35383506/what-is-the-difference-between-encoding-utf-8-and-utf8-in-python-3-5

Reply:
yes, It is just alias. 
You updated to `utf-8` in other place in this PR,  I thought you want to update all to keep consistency.

Reply:
Updated!